src/styles/_fonts.scss

```diff
@@ -4,3 +4,5 @@
 @import '~fontsource-ibm-plex-sans/500-normal.css';
 @import '~fontsource-ibm-plex-sans/700-normal.css';
 @import '~fontsource-ibm-plex-sans/700-italic.css';
+
+@import '~typeface-roboto/index.css';
```

Comment:
@onvisions is it correct that we need to add a new font specially for this error popup?